Tasks/GooglePlayReleaseV4/modules/metadataHelper.ts

```diff
@@ -61,27 +61,49 @@ async function addAllReleaseNotes(versionCodes: number[], languageCode: string,
     }
 
     const releaseNotes: pub3.Schema$LocalizedText[] = [];
+    let fullDefaultChangelogPath: string = '';
     for (const changelogFile of changelogs) {
         const changelogName: string = path.basename(changelogFile, path.extname(changelogFile));
         const changelogVersion: number = parseInt(changelogName, 10);
+        if (changelogName === 'default') {
+            fullDefaultChangelogPath = path.join(changelogDir, changelogFile);
```

Comment:
I'd add `continue` not to write the warning `The name of the file default.txt is not a valid version code. Skipping it` for default file

Reply:
Done. Thank you for the suggestion.